server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/DomainsRoutes.java

```diff
@@ -156,9 +178,59 @@ public void defineGetDomains() {
             jsonTransformer);
     }
 
+    @GET
+    @Path("/{domainName}/aliases")
+    @ApiOperation(value = "Getting all aliases for a domain")
+    @ApiImplicitParams({
+        @ApiImplicitParam(required = true, dataType = "string", name = "domainName", paramType = "path")
+    })
+    @ApiResponses(value = {
+        @ApiResponse(code = HttpStatus.OK_200, message = "OK", response = List.class),
+        @ApiResponse(code = HttpStatus.NOT_FOUND_404, message = "The domain does not exist."),
+        @ApiResponse(code = HttpStatus.INTERNAL_SERVER_ERROR_500,
+            message = "Internal server error - Something went bad on the server side.")
+    })
+    public void defineListAliases(Service service) {
+        service.get(DOMAIN_ALIASES, this::listDomainAliases, jsonTransformer);
+    }
+
+    @DELETE
+    @Path("/{sourceDomain}/aliases/{destinationDomain}")
+    @ApiOperation(value = "Remove an alias for a specific domain")
+    @ApiImplicitParams({
+        @ApiImplicitParam(required = true, dataType = "string", name = "sourceDomain", paramType = "path"),
+        @ApiImplicitParam(required = true, dataType = "string", name = "destinationDomain", paramType = "path")
+    })
+    @ApiResponses(value = {
+        @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "OK", response = List.class),
+        @ApiResponse(code = HttpStatus.NOT_FOUND_404, message = "The domain does not exist."),
+        @ApiResponse(code = HttpStatus.INTERNAL_SERVER_ERROR_500,
+            message = "Internal server error - Something went bad on the server side.")
+    })
+    public void defineRemoveAlias(Service service) {
+        service.delete(SPECIFIC_ALIAS, this::removeDomainAlias, jsonTransformer);
+    }
+
+    @PUT
+    @Path("/{sourceDomain}/aliases/{destinationDomain}")
```

Comment:
I think it should be `"/{destinationDomain}/aliases/{sourceDomain}"` where the destination domain represents for the subject where the source domain represents for the alias.

Reply:
I did do the mistake in the routes and catched it in the unit test but forgot to turn it around in the doc...